app/views/devise/registrations/new.html.slim

```diff
@@ -4,9 +4,9 @@
     = f.govuk_fieldset legend: { text: 'Create an early years training account', tag: :h1, size: 'l' } do
       p.govuk-body
         | Create an account to access training courses.
-      = f.govuk_email_field :email, autofocus: true, autocomplete: "email", label: { text: "Email address" }, width: 'one-half'
-      = f.govuk_password_field :password, autocomplete: "new-password", label: { text: "Create password" }, width: 'one-half', hint: { text: "(#{@minimum_password_length} characters minimum)" }
-      = f.govuk_password_field :password_confirmation, autocomplete: "new-password", label: { text: "Re-type password" }, width: 'one-half'
+      = f.govuk_email_field :email, autofocus: true, autocomplete: "email", label: { text: "Email address" }, width: 'two-thirds', hint: { text: "We'll send you an email with an account" }
```

Comment:
**"We'll send you an email with an account"** - what does that mean?

Should it be "We'll send an email to your account"?

And should this be defined in a translation file?

Reply:
Yea, that makes more sense - but I think 'account' here means the account on the site, not your email account.